service/etcdv2/spec.go

```diff
@@ -0,0 +1,13 @@
+package etcdv2
+
+import (
+	"context"
+)
+
+type Store interface {
```

Comment:
For testing we need to configure an interface. We can then replace it with the fake implementation.